index.html

```diff
@@ -95,17 +95,17 @@
   
   <main id="main-content">
     
-  <div id="breadcrumbs">
+  <div id="breadcrumbs" aria-label="navigation breadcrumb">
     <nav class="breadcrumb is-centered is-medium" role="breadcrumb">
       <ul>
-        <li><a href="./" class="current" aria-current="page">MainPage</a><span></span></li>
+        <li><a href="./" class="current" aria-current="page" aria-selected="true">MainPage</a><span></span></li>
         <li><a href="index.html#nav">Pictures</a><span></span></li>
         <li><a href="index.html#services-section">Services</a></li>
       </ul>
     </nav>
   </div>
 
-  <section class="section" id="about" aria-labelled="about-section">
+  <section class="section" id="about" aria-label="about-section">
```

Comment:
Could you please add `role="table"` in the line 171

<img width="413" alt="image" src="https://user-images.githubusercontent.com/16785774/172821723-ea4e14df-96d9-49f6-b481-6b21ad964126.png">

Reply:
...also `role="columnheader"` for **<th** tags line 174, 175

<img width="230" alt="image" src="https://user-images.githubusercontent.com/16785774/172822225-3f50287f-ed25-4229-a216-20968d616703.png">